pilot/prompts/development/iteration.prompt

```diff
@@ -13,7 +13,7 @@ Here are the technologies that you need to use for this project:
 {% if development_tasks and current_task %}
 Development process of this app was split into smaller tasks. Here is the list of all tasks:
 ```{% for task in development_tasks %}
-- {{ loop.index }}. {{ task['description'] }}
+{{ loop.index }}. {{ task['description'] }}
```

Comment:
Nitpick: I fail to see what needed a change here :)

Reply:
Just to be consistent with other parts of the codebase